contract/src/contract.rs

```diff
@@ -134,15 +131,64 @@ pub enum HandleMsg {
     Fold {},
     Check {},
     Rematch {},
+    Withdraw {},
 }
 
+pub fn winner_winner_chicken_dinner(contract_address: HumanAddr, player: HumanAddr) -> HandleResponse {
+    HandleResponse{
+        messages: vec![CosmosMsg::Bank(BankMsg::Send {
+            from_address: contract_address,
+            to_address: player,
+            amount: vec![Coin{
+                denom: "uscrt".to_string(),
+                amount: Uint128(200_000_000),
+            }]}),
+        ],
+        log: vec![],
+        data: None
+    }
+}
+
+
 pub fn handle<S: Storage, A: Api, Q: Querier>(
     deps: &mut Extern<S, A, Q>,
     env: Env,
     msg: HandleMsg,
 ) -> HandleResult {
     match msg {
+        HandleMsg::Withdraw {} => {
+
+            let player_name = deps.api.human_address(&env.message.sender)?;
+            let contract_address = deps.api.human_address(&env.contract.address)?;
+
+            let mut table: Table =
+                serde_json::from_slice(&deps.storage.get(b"table").unwrap()).unwrap();
```

Comment:
I believe that you can use `?` for the `storage.get` and `serde_json::from_slice` can be replaced with `cosmwasm_std::from_slice` which also can be used with `?` here.